target/linux/generic/hack-6.6/820-ath9k-add-support-for-registering-extra-leds-connect.patch

```diff
@@ -0,0 +1,27 @@
+From 956f24ebb7b7733416e5f194db98a989f1c5ad9c Mon Sep 17 00:00:00 2001
+From: OpenWrt community <openwrt-devel@lists.openwrt.org>
```

Comment:
You can find proper authors and commits messages etc in the patches in `package/kernel/mac80211` that make use of the struct members.